Test/Algo2023Day08Tests/NodeBuilderTest.cs

```diff
@@ -0,0 +1,73 @@
+﻿using FluentAssertions;
+using Lib.Day07Year2023;
+using Lib.Day08Year2023;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Test.Algo2023Day08Tests
+{
+    public class NodeBuilderTest
+    {
+        [Fact]
+
+        public void NodeNameListBuilder_GivenCorectInput_ShouldBeCorectValue()
+        {
+            // Given           
+            NodeBuilder nodeBuilder = new NodeBuilder();
+
+            // When
+            List<Node> actual = nodeBuilder.NodeNameListBuilder(["AAA = (BBB, BBB)","BBB = (AAA, ZZZ)","ZZZ = (ZZZ, ZZZ)"]);           
+
+            // Then
+            actual.Should().NotBeEmpty();
+            actual.Count.Should().Be(3);
+            actual[0].Name.Should().BeEquivalentTo("AAA");
```

Comment:
d'après le nom, on ne veut tester que ça

```suggestion
            actual[0].Name.Should().BeEquivalentTo("AAA");
```